openstack_controller/tox.ini

```
@@ -3,7 +3,7 @@ minversion = 2.0
 skip_missing_interpreters = true
 basepython = py37
 envlist =
-    py{27,37}-openstack_controller
+    py{27,37}-openstack_controller-{unit,integration}
```

```suggestion
    py{27,37}-{unit,integration}
```
Similar to how we do it with others